VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionPostsActivity.java

```diff
@@ -63,9 +69,20 @@ protected void onStart() {
         }
 
         if (discussionTopic != null && discussionTopic.getName() != null) {
-            setTitle(discussionTopic.getName());
+            if (discussionTopic.getIdentifier().equals(DiscussionTopic.FOLLOWING_TOPICS_ID)) {
+                SpannableString title = new SpannableString("  " + discussionTopic.getName());
+                Drawable starIcon = new IconDrawable(this, FontAwesomeIcons.fa_star)
+                        .colorRes(this, R.color.edx_grayscale_neutral_white_t)
+                        .sizeRes(this, R.dimen.edx_base);
+                starIcon.setBounds(0, 0, starIcon.getIntrinsicWidth(), starIcon.getIntrinsicHeight());
```

Comment:
Unsure why there is a tint set, but setting the tint to null will remove the tint.

starIcon.setTintList(null);